project/AppDependencies.scala

```diff
@@ -6,7 +6,7 @@ object AppDependencies {
     "uk.gov.hmrc"             %% "play-language"            % "4.3.0-play-26",
     "uk.gov.hmrc"             %% "bootstrap-play-26"        % "1.14.0",
     "uk.gov.hmrc"             %% "play-frontend-govuk"      % "0.49.0-play-26",
-    "uk.gov.hmrc"             %% "play-frontend-hmrc"       % "0.16.0-play-26",
+    "uk.gov.hmrc"             %% "play-frontend-hmrc"       % "0.17.0-play-26",
     "com.typesafe.play"       %% "play-json-joda"           % "2.6.13",
```

Comment:
Is this library still required?

Reply:
yes

Reply:
Sorry joda, yes that can be removed. done